src/envoy/http/jwt_auth/jwt_authenticator.cc

```diff
@@ -206,21 +207,11 @@ void JwtAuthenticator::VerifyKey(const PubkeyCacheItem& issuer_item) {
 }
 
 bool JwtAuthenticator::OkToBypass() {
```

Comment:
What this bypass logic is about? Is this removed from the new API, or you just delay the implementation? If the former, just remove this function completely. Otherwise, add some comment why it's not needed yet. In fact, if it's not needed for common use cases, delete it still. We can wait to have it back with upstream version.